### DISCUSSION TITLE
Apologies for what seems like a basic question

### DISCUSSION THREAD

**Discussion by Jeremy-MW** [keirf/flashfloppy#621] [Q&A]
- I cannot find the answer to this simple question anywhere:

WHY is FlashFloppy needed?
What does it give you (for regular PC use, or in my case a Yamaha A3000 sampler) that the out-of-the-box GOTEK doesn't?

My basic, stock GOTEK worked fine straight off in the A3k without needing replacement 3rd party software so I'm a bit baffled. The only thing I cannot seem to do is read the resulting USB stick in a PC (after using in the A3k, the USB stick doesn't show up when plugged into a PC).

Is THAT what FlashFloppy gives you? The ability to create a USB stick that's readable on a PC?
How does one read/write to a GOTEK virtual floppy USB stick plugged directly into a PC?

Thanks in advance - this information (if out there) must be buried under all the posts about mods etc, because I can't find it...

**Comment by keirf** [comment]
Yes in a way. FlashFloppy works with IMG files on the USB stick. Each of those IMG files is a virtual disk image. You can see them in Windows but you can't see directly *inside* them. For that you need something that understands virtual disk images, like WimImage or (suprisingly) many archive tools such as WinZip.

**Reply by Jeremy-MW** [reply]
OK - But this is equally possible out-of-the-box with a Gotek, with USB Floppy Manager Batch Manage Tool. 

Also, out-of-the-box with the Gotek, disk zero is just the regular root folder of the USB drive (which is convenient), and so an image isn't needed for that one, and the floppy manager is really easy-to-use to get things into/out of up to 100 images.

So I am still struggling to see the benefit of FlashFloppy?

**Reply by keirf** [reply]
For basic out-of-the-box emulation of one or two IBM-PC disks, the factory firmware is perfectly fine.

**Comment by chrispoacher** [comment]
On a 16gb usb key the basic gotek will still only give you 100 images. Flashfloppy gives you about 10,000 with long filenames,  folders/sub-folder navigation with oled screen fitted. No brainer really :)

Chris

**Reply by Jeremy-MW** [reply]
Actually, it's a no-brainer for me NOT to use FlashFloppy 😃

I can see now that FlashFloppy makes sense for retro gaming, where you need to have the data stored on a virtual floppy because that's where the game expects to find its disks.

With my Yamaha A3000 sampler however the situation is different. You don't want anything in storage on virtual floppy at all (so, no need for a rotary dial or deluxe LCD). The Gotek is simply needed because the A3000 sample export function is unfortunately hard-wired to use the floppy drive only,

Having such samples exported onto the root folder of a USB stick is perfect (to immediately be copied elsewhere), and so it's clear that I don't need FlashFloppy. I've been testing things out over the last 24 hours, and the factory functionality suits my needs just fine.

Thanks everyone for your answers, you've been very helpful.

**Comment by Dietrich-L** [comment]
The big advantage comes, if you need non-IBM-PC disk formats. Those are needed in many vintage computers and computer controlled equipment, where you replace old floppy drives of various formats - 8“, 5.25“, 3.5“, 3“ and more, all with shugart compatible interfaces. In these applications FlashFloppy is invaluable.

Dietrich

**Comment by Jeremy-MW** [comment]
Thanks everyone - as per my reply to Chris above, I don't need FlashFloppy.

**Comment by chrispoacher** [comment]
I can see how it would be useful if you are ‘only’ going to use the A3000 for that purpose of exporting to wav. Just putting the usb key drive in the pc always activates the first image as drive A (or B if you already have a floppy drive). This is pretty much the exception to the rule though and just useful for A3000 I’m guessing.  However I’d still use it in mine and save multiple disks at a time and use HXC software to extract from the images, but that’s just me. :) if you have V2 roms in the A3000 you are not limited to a single disk to save your programs and samples like the V1 roms which are pretty buggy.

**Reply by Jeremy-MW** [reply]
Not sure what you mean by drives A and B? I don’t have a Gotek attached to the PC, just the one mounted inside the A3000. 

On the PC I just look at the root of the USB stick as per any other USB stick. 

Do you know of a way to stitch together long samples saved across multiple floppies, along with the loop points?

That would certainly be useful to me. In the absence of preserving loop points there’s no point using the Gotek - I just send the very long samples over SPDIF to the PC, chop them up and loosely recreate the original loop points by hand. 

Do you know of a way of extracting long samples from the partial files created across multiple floppies?

**Comment by diodep** [comment]
Hello and maybe out of topic...

Can you please share the image file of original YAMAHA A3000 factory disks (about 6 disks)?

I want this to bring my second-handed EX5 to live.

Thank you!
